roles/devscripts/tasks/300_post.yml

```diff
@@ -31,11 +31,12 @@
     - devscripts_deploy
   when:
     - not cifmw_devscripts_ocp_comply | bool
+    - cifmw_use_ocp_overlay | default(false) | bool
   ansible.builtin.include_tasks: 310_prepare_overlay.yml
 
 - name: Bringing cluster online.
   when:
-    - not cifmw_use_reproducer | default(false) | bool
+    - cifmw_use_ocp_overlay | default(false) | bool
   block:
```

Comment:
no need for a block if you have only one task nested there

Reply:
Done